msticpy/nbtools/timeseries.py

```diff
@@ -8,6 +8,8 @@
 from itertools import zip_longest
```

Comment:
Can you extract this into a separate module and put in the analysis folder? I want to try to move things out of nbtools and this seems like a good can

Reply:
I can also add my time region extraction functions to this.

Reply:
moved function under analysis root.